### ISSUE TITLE
Feature Request - Multiple Apple ID accounts

### ISSUE THREAD

**Issue by jwoglom** [gcobb321/icloud3_v3#222]
Can icloud3 be used with multiple logged-in Apple IDs, similar to the native Home Assistant Apple iCloud integration? It seems that the configuration only allows having one active apple ID logged in at a time. Is it possible to work around this? (e.g. could I duplicate the `custom_components/icloud3` folder, or would that break everything?)

**Comment by gcobb321** [comment]
No, iCloud3 only supports one account. Also, only one instance of iCloud3 can be added to HA so creating another directory and trying to run it from 2 directories will not work either. 

I’ve changed the heading of this issue to a feature request

**Comment by Aekung** [comment]
I would like to have this feature as well as single iCloud ID doesn't really fit my use case. Do you have any update on this feature?

Thank you and looking forward to hearing from you.

**Comment by gcobb321** [comment]
I’ve been concentrating on getting v3 wrapped up and haven’t given it any thought. I would like to find a solution to getting access to Find My Friends location data before tackling the issue.

**Comment by jspeckman** [comment]
I am also interested in the feature.
As a work around for any interested, since I was already running home assistant in a container, it was fairly easy to spin up an additional container to run another home assistant install with a separate icloud3 integration.  I connected the new install to my primary one with the remote integration.  It's a little bit of a kluge, but it does work.  To get zone names to show up, zones need to be kept in sync between the home assistant instances.

**Comment by gcobb321** [comment]
Closing due to no activity and the release of iCloud3 v3.1.5
